src/Components/Open/SaveModelControl.jsx

```diff
@@ -123,6 +123,7 @@ function SaveModelDialog({isDialogDisplayed, setIsDialogDisplayed, navigate, org
           repoName,
           'main',
           accessToken,
+          isOpfsAvailable,
```

Comment:
Isn't this shadowing the same variable in the outer scope?  If you want to keep it as a param then rename